.github/workflows/release.yml

```
@@ -1,4 +1,7 @@
-name: MushroomSoft Release and Deploy
+name: MushroomSoft Release and Deploy Production
+description: |
```

actionlint (1.7.7)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)